sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -435,8 +435,11 @@ protected void addFields() {
 
 		// Set initial visibilities & accesses
 		initializeVisibilitiesAndAllowedVisibilities();
-
-		setRequired(true, CaseDataDto.REPORT_DATE, CaseDataDto.DISEASE, FACILITY_OR_HOME_LOC, FACILITY_TYPE_GROUP_LOC, CaseDataDto.FACILITY_TYPE);
+		
+		setRequired(true, CaseDataDto.REPORT_DATE, CaseDataDto.DISEASE, FACILITY_TYPE_GROUP_LOC, CaseDataDto.FACILITY_TYPE);
+		if (!UserProvider.getCurrent().isPortHealthUser()) {
```

Comment:
this condition can be found in the class twice. Please add your changes under the other already existing condition.